### PR TITLE
Added Sleep as Android icon (closes #24)

### LINKED ISSUE #24: Sleep As Android (SVG included)
App Name: Sleep As Android
Package: com.urbandroid.sleep
Icon (SVG):
[sleep_as_android.txt](https://github.com/LawnchairLauncher/lawnicons/files/7490421/sleep_as_android.txt)

### PR DESCRIPTION
**Pull request by hardeik**

### CODE REVIEW

File: svgs/sleep_as_android.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	 viewBox="0 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill-rule:evenodd;clip-rule:evenodd;}
+</style>
+<path class="st0" d="M5,2L3.9,3.1l2.7,2.7C4.3,7.5,3,10.2,3,13v1h18v-1c0-2.8-1.3-5.5-3.6-7.2l2.7-2.7L19,2l-2.9,3
+	c-2.6-1.3-5.7-1.3-8.3,0L5,2z M12,5c-4.4,0-8,3.6-8,8h16C20,8.6,16.4,5,12,5z M8.1,11.3c0.9,0,1.7-0.5,2-1.3c0,1.1-0.9,2-2,2
+	s-2-0.9-2-2C6.5,10.7,7.2,11.3,8.1,11.3z M13.9,10c0.5,1.1,1.8,1.6,2.9,1.1c0.5-0.2,0.9-0.6,1.1-1.1c0,1.1-0.9,2-2,2
+	S13.9,11.1,13.9,10z M21,15.1l-1,0.4V21h-2v-4.8l-1.5,0.6V21h-2v-3.3l-1.2,0.5L13,17.6V21h-2v-3.4l-0.3,0.5l-1.2-0.5V21h-2v-4.1
+	L6,16.2V21H4v-5.6l-1-0.4v7h18C21,22,21,15,21,15.1z M10.5,17.5l-7.5-3h9L10.5,17.5z M21,14.5l-7.5,3l-1.5-3H21z"/>
+</svg>
```
Comment (patrykmichalik) [inline svgs/sleep_as_android.svg:13]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   fill="none"
   version="1.1"
   id="svg4"
   sodipodi:docname="sleep_as_android.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="16.1875"
     inkscape:cx="-2.9034749"
     inkscape:cy="5.5907336"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4" />
  <g
     id="g829"
     transform="matrix(0.16393377,0,0,0.16393377,2.8197089,2.00008)"
     style="fill:#000000">
    <path
       fill-rule="evenodd"
       clip-rule="evenodd"
       d="m 15.0156,-5.5727232 c 1.918,-1.09619 4.3614,-0.42968 5.4575,1.4882799 L 30.751,13.901881 C 38.3374,10.062981 46.916,7.8999793 56,7.8999793 c 9.084,0 17.6626,2.1630017 25.2495,6.0019017 L 91.5273,-4.0844433 c 1.0957,-1.9179599 3.5396,-2.5844699 5.4576,-1.4882799 1.9179,1.0957099 2.5844,3.53955 1.4882,5.45751998 L 88.1123,18.016181 C 102.557,28.144081 112,44.919481 112,63.899981 v 8 36.000019 8 H 104 7.9999999 -1.2200042e-7 v -8 -36.000019 -8 C -1.2200042e-7,44.918981 9.4433599,28.144081 23.8882,18.016181 L 13.5273,-0.11520322 C 12.4312,-2.0331732 13.0977,-4.4770133 15.0156,-5.5727232 Z M 56,15.899981 c 26.5098,0 48,21.4902 48,48 H 92 84 71 63 49 41 28 20 7.9999999 c 0,-26.5098 21.4902001,-48 48.0000001,-48 z m 10,74 -10,-18 -10,18 -38.0000001,-14.8697 V 107.9 H 20 V 87.726081 l 8,3.1304 V 107.9 H 41 V 95.942986 l 5,1.957006 3,1.200004 V 107.9 h 14 v -8.800004 l 3,-1.200004 5,-1.957006 V 107.9 H 84 V 90.856481 l 8,-3.1304 V 107.9 h 12 V 75.030281 Z m -33,-39 c 6.3247,0 11.5942,-4.5166 12.7598,-10.5 0.083,0.4253 0.145,0.8584 0.1855,1.2973 0.0361,0.396 0.0547,0.7974 0.0547,1.2027 0,7.1796 -5.8203,13 -13,13 -7.1797,0 -13,-5.8204 -13,-13 0,-0.855 0.0825,-1.691 0.2402,-2.5 1.1656,5.9834 6.4351,10.5 12.7598,10.5 z m 58.7598,-10.5 c -1.1656,5.9834 -6.4351,10.5 -12.7598,10.5 -6.3247,0 -11.5942,-4.5166 -12.7598,-10.5 -0.1577,0.809 -0.2402,1.645 -0.2402,2.5 0,7.1796 5.8203,13 13,13 7.1797,0 13,-5.8204 13,-13 0,-0.4683 -0.0249,-0.9312 -0.0732,-1.3868 -0.0401,-0.3759 -0.0957,-0.7475 -0.167,-1.1132 z"
       fill="#000000"
       id="path1068"
       style="fill:#000000;stroke-width:0.999999" />
  </g>
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Here are some tweaks I’d suggest.

**Comment by hardeik** [comment]
@patrykmichalik Done.

**Comment by patrykmichalik** [comment]
Thanks for contributing, @hardeik!

**Pull request merged**